### DISCUSSION TITLE
Use an error monitoring tool

### DISCUSSION THREAD

**Discussion by francoiscrouzet** [archaiodata/thacer#20] [Ideas]
Like sentry

**Comment by AlZorba** [comment]
I will have a look. BTW why don't we merge this discussion with technologies we may use?

**Reply by francoiscrouzet** [reply]
Hum, je dirais pour 2 raisons
- les technologies sont des librairies/frameworl pour developpeurs, alors que sentry est un service externe qu'on peut utiliser directement
- si ça te va, je risque d'installer sentry "par plaisir" prochainement. Alors que pour la liste de technologie, je pense que c'est pour plus tard, quand on en aura besoin

**Comment by francoiscrouzet** [comment]
Si ça te va, je vais l'installer sur mon temps libre. En mode test/freemium a droit à un seul compte, donc je propose de nous creer une addresse mail `archaiodata@gmail.com` par exemple?
Ok pour les 2?
